.env.development

```diff
@@ -1,3 +1,4 @@
+REACT_APP_NEW_TEACHER_SITE=false
```

Comment:
the .env.development file is ignored now.
can you remove it from the commit pls? 

AH, and please send the PR to development not master!

Reply:
Ah! Sorry for being sloppy and sending the PR to the wrong branch. I wondered why all old commits were included into this PR - makes sense now. I'll pay much more attention next time.